packages/ui/components/ui/network.tsx

```diff
@@ -0,0 +1,38 @@
+import { ExternalLink } from 'lucide-react';
+import { ConditionalWrap } from './conditional-wrap';
+
+export interface NetworkProps {
+  name: string;
+  connectIndicator?: boolean;
+  href?: string;
+}
+
+/**
+ * Displays the network (chain ID) we're connected to, as well as an optional
+ * connection indicator and a link to the frontend.
+ */
+export const Network = ({ name, href, connectIndicator = true }: NetworkProps) => {
```

Comment:
This was previously `NetworksPopover`

Reply:
I ran the extension first and only then read the PR. 
Actually, it was not very obvious to me that clicking on network should lead to minifront (I actively use other wallet extensions).
I think the closest analogy for minifront is the metamask portfolio or keplr portfolio, and it would probably be familiar to me if the transition to minifront was similar. 

At the same time, it would be predictable for me if clicking on network would lead to the network status page. 
https://grpc.testnet.penumbra.zone

<img width="370" alt="image" src="https://github.com/penumbra-zone/web/assets/25391690/30aebf32-c3f9-4bf2-8435-db129edf75db">
<img width="367" alt="image" src="https://github.com/penumbra-zone/web/assets/25391690/d1d2b060-29e4-45d6-91f9-9e68e21c8eab">

Reply:
@Valentine1898 Great feedback — I'll whip something together for this and re-request review

Reply:
@Valentine1898 I've redesigned it like this:

![image](https://github.com/penumbra-zone/web/assets/1121544/88a91e07-c514-4416-8f69-96c12f4ce5b3)